flink-cdc-cli/src/main/java/org/apache/flink/cdc/cli/parser/YamlPipelineDefinitionParser.java

```diff
@@ -188,7 +189,14 @@ private SourceDef toSourceDef(JsonNode sourceNode) {
         // "name" field is optional
         String name = sourceMap.remove(NAME_KEY);
 
-        return new SourceDef(type, name, Configuration.fromMap(sourceMap));
+        // "parallelism" field is optional
+        Integer parallelismValue = null;
```

Comment:
Could we use `pipeline.parallelism`'s valued as  default parallelism instead of an error-prone null value? then we  can use `int` type and do not need to use `@Nullable ` annotation in related methods.